export_results.py

```diff
@@ -495,29 +503,200 @@ def export_updated_phenos(num_pops=None):
                        skip_binary_eur=False)
 
 
-def make_pheno_manifest(export=True):    
+def make_per_population_n_cases():
+    """
+    This function computes per-population and per-sex case count for each phenotype.
+    """
+    path_loc = f'{bucket}/combined_results/per_pop_per_sex_case_counts.ht'
+    ht_saige = hl.read_matrix_table(get_variant_results_path('full')).cols()
+    ht_saige = ht_saige.explode('pheno_data')
+    ht_saige = ht_saige.annotate(pop = ht_saige.pheno_data.pop).key_by(*PHENO_KEY_FIELDS, 'pop')
+
+    if hl.hadoop_exists(path_loc):
+        ht_per_pop = hl.read_table(path_loc)
+    else:
+        samples_keep = get_filtered_mt(chrom='22').cols()
+        mt = hl.read_matrix_table(get_ukb_pheno_mt_path())
+        mt = mt.annotate_rows(s = hl.str(mt.userId)).key_rows_by('s')
+        mt = mt.semi_join_rows(samples_keep)
```

Comment:
Can we get some sanity checking counts here (maybe per pop too)?

Reply:
done